Lint检查失败: feishu-channel-passive-mode.test.ts 使用对象解构

## 测试输出错误信息

\`\`\`
/Users/hs3180/clawd/disclaude/src/channels/feishu-channel-passive-mode.test.ts
  142:9  error  Use object destructuring  prefer-destructuring
\`\`\`

## 根本原因分析

在 PR #518 (commit 52f4969) 中新增的测试文件 \`src/channels/feishu-channel-passive-mode.test.ts\` 第 142 行：

\`\`\`typescript
let chatType = options.chatType;
\`\`\`

ESLint 的 \`prefer-destructuring\` 规则要求使用对象解构而不是属性访问。应该改为：

\`\`\`typescript
let { chatType } = options;
\`\`\`

或者如果需要重新赋值 chatType，考虑使用不同的变量名。

## 相关提交

- Commit: 52f4969 - fix(feishu): don't add reaction to skipped messages in passive mode (Issue #514) (#518)
- 修改文件: \`src/channels/feishu-channel-passive-mode.test.ts\`

## 修复建议

运行 \`npm run lint:fix\` 可以自动修复此问题，或手动修改第 142 行使用对象解构。

🤖 Generated by CI Monitor scheduled task

--- a/src/channels/feishu-channel-passive-mode.test.ts
+++ b/src/channels/feishu-channel-passive-mode.test.ts
@@ -139,7 +139,7 @@ describe('FeishuChannel - Group Chat Passive Mode (Issue #460)', () => {
     mentions?: Array<{ key: string; id: { open_id: string }; name: string }>;
   }): Promise<void> {
     // Determine chat_type based on chatId prefix if not explicitly provided
-    let chatType = options.chatType;
+    let { chatType } = options;
     if (!chatType) {
       const chatId = options.chatId || 'oc_test_group';
       if (chatId.startsWith('oc_')) {

